benchmarks/samples/fabric/fabcar/queryAllCars.js

```diff
@@ -0,0 +1,44 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+module.exports.info = 'Querying all cars.';
+let startIndex = 1;
+let endIndex = 50;
+let bc, contx;
+
+module.exports.init = function (blockchain, context, args) {
+    bc = blockchain;
+    contx = context;
+
```

Comment:
We should probably place some "sample cars" in here to make sure that there is something to query if this is run in isolation. They will over-write any existing items if there is an index conflict, so no need to worry. You can also have a "create cars" flag that is passed as an argument to conditionally create the cars if it is run in isolation too (this is done in the API benchmark)